internal/lg/lg.go

```diff
@@ -0,0 +1,62 @@
+// short for "log"
+package lg
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Logger interface {
+	Output(maxdepth int, s string) error
+}
+
+const (
+	DEBUG = 1
+	INFO  = 2
+	WARN  = 3
+	ERROR = 4
+	FATAL = 5
+)
+
+func ParseLogLevel(levelstr string, verbose bool) (int, error) {
```

Comment:
Here's what I had in mind: https://play.golang.org/p/bUb1ETf4zY

What do you think?